rasa_nlu/data_router.py

```diff
@@ -210,7 +235,9 @@ def start_train_process(self, data, config_values):
             _config[key] = val
         _config["data"] = f.name
         train_config = RasaNLUConfig(cmdline_args=_config)
-        process = multiprocessing.Process(target=do_train, args=(train_config, self.component_builder))
-        self._add_train_proc(process)
```

Comment:
We could. I removed it completely because I wanted to track the trainings at the endpoint level (that was also why I proposed the "agents" change with Klein in the first place). Now we have processes ids. This is really low-level information. However I can reintroduce the actual training tracking queue but as the default behavior is to have [one dedicated process for training](https://github.com/RasaHQ/rasa_nlu/pull/450#discussion_r126166737), is that necessary? What do you think?